prediction_market_agent_tooling/deploy/betting_strategy.py

```diff
@@ -443,8 +443,22 @@ def calculate_trades(
             kelly_bet_size = min(kelly_bet.size, max_price_impact_bet_amount)
 
         bet_outcome = direction if kelly_bet.direction else other_direction
+
+        kelly_bet_size_usd = market.get_token_in_usd(kelly_bet_size)
+        existing_amount = (
+            existing_position.amounts_current.get(bet_outcome, USD(0))
+            if existing_position
+            else USD(0)
+        )
+        capped_amount = BettingStrategy.cap_to_profitable_position(
+            market=market,
+            existing_position=existing_amount,
+            wanted_position=existing_amount + kelly_bet_size_usd,
```

Comment:
```suggestion
            wanted_position=kelly_bet_size_usd,
```

Shouldn't this be just like this?

Otherwise, it's always increasing the `max_position_amount` by the previous bet, which isn't what is wanted. (just a design decision that we agreed on, but now it's nice to have it consistent between strategies).